recipes/wake-t/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "0.7.1" %}
+
+package:
+  name: wake-t
+  version: {{ version }}
+
+source:
+  url: https://github.com/AngelFP/Wake-T/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9386cc52ca98e272518686da3e02a690363ecde689f7b03b07aeb2ecece2ee05
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - aptools
+    - numba
+    - numpy
+    - openpmd-api
+    - python >=3.6
+    - scipy
+    - tqdm
+
+test:
+  requires:
+    - matplotlib
```

Comment:
```suggestion
    - matplotlib-base
```

Reply:
Sure, thx! I think since this is only in the test section, it is no real issue to use the full package because users do not need to switch it.